pkg/api/convertertoapi.go

```diff
@@ -382,6 +382,11 @@ func convertVLabsProperties(vlabs *vlabs.Properties, api *Properties) {
 		api.CertificateProfile = &CertificateProfile{}
 		convertVLabsCertificateProfile(vlabs.CertificateProfile, api.CertificateProfile)
 	}
+
```

Comment:
I think we need all this conversion logic in `converterfromapi.go` in the reverse direction as well.